contracts/optimistic-ethereum/iOVM/execution/iOVM_ExecutionManager.sol

```diff
@@ -11,6 +11,7 @@ interface iOVM_ExecutionManager {
      *********/
 
     enum RevertFlag {
+        // UNUSED, // Save gas by not setting a storage slot to zero
```

Comment:
Added this here for discussion. 
Currently it breaks all the revert flag tests, but they shouldn't be too hard to fix once I know what's going on in there.

Reply:
ah yes, there is an off-chain revert flag enum somewhere in like `test-helpers` that will need to be updated.